wire-schema/src/main/java/com/squareup/wire/schema/MarkSet.java

```diff
@@ -38,7 +38,8 @@
 final class MarkSet {
   final IdentifierSet identifierSet;
   final Set<ProtoType> types = new LinkedHashSet<>();
-  final Multimap<ProtoType, ProtoMember> members = LinkedHashMultimap.create();
+  final Multimap<ProtoType, ProtoMember> protoMembers = LinkedHashMultimap.create();
+  private final Set<String> members = new LinkedHashSet<>();
```

Comment:
I don’t think this is a sufficiently narrow namespace? What are the entries in this set?

Reply:
I agree, I thought about that too this morning